src/components/header/Header.tsx

```diff
@@ -94,7 +100,41 @@ const Header = (props: Props) => {
         >
           Games
         </Link>
+        {user?.data ? (
+          <NavDropdown
+            title={<BiSolidUser size={30} />}
+            id="navbarScrollingDropdown"
+          >
+            <NavDropdown.Item>
+              <Link
+                style={{ color: 'gray' }}
+                onClick={() => window.scrollTo(0, 0)}
+                href={`/profile`}
```

Comment:
Why bacsticks with {}? We dont have any dynamic variables in href and it would be better to use ""